### ISSUE TITLE
cant launch link

### ISSUE THREAD

**Issue by nitishpatel** [sur950/any_link_preview#6]
- any_link_preview: ^1.0.6
 
Cant Launch Link
```
E/flutter (14536): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: Could not launch https://www.recruitinghub.com/
E/flutter (14536): #0      _AnyLinkPreviewState._launchURL (package:any_link_preview/any_link_preview.dart:147:7)
E/flutter (14536): <asynchronous suspension>
```

**Comment by sur950** [comment]
Hi @nitishpatel 

Thanks for using the package. I have tried with your link ```https://www.recruitinghub.com/``` and I am using `url_launcher` package as internal dependency. In which, we had a method to check whether the specified URL can be launched or not using ```canLaunch()``` method, which is giving false. This is the reason why it is throwing error.

However, I will add one more check in order to make the flow more smooth. Once I have released new version, I will let you know.

Thanks!
